doc/report-issue.md

```diff
@@ -0,0 +1,37 @@
+## Report an issue
+
+This document outlines how to report an issue for the static analyzer and
+ensure that your bug report is complete with all required information.
+
+> [!TIP]
+> If you are a Datadog customer, send the bug report with all information
+> to your customer success manager.
```

Comment:
"If you are a Datadog customer in contact with your account team, send your bug report with all information to your contact(s)."